system/mp/mpcomm.cpp

```diff
@@ -853,10 +853,10 @@ protected: friend class CMPPacketReader;
 #if defined(_USE_OPENSSL)
                 if (parent->useTLS)
                 {
+                    Owned<ISecureSocket> ssock = secureContextClient->createSecureSocket(newsock.getClear());
                     int tlsTraceLevel = SSLogMin;
-                    if (parent->mpTraceLevel)
+                    if (parent->mpTraceLevel > DebugMsgThreshold)
```

Comment:
Should this be testing if > MyVerboseLevel (which is higher than the default for now), to suppress extra tracing?